runtime/src/main/resources/application.yml

```diff
@@ -80,3 +80,11 @@ generator:
 controllers:
   integration:
     enabled: true
+
+client:
+  state:
+    authenticationAlgorithm: HmacSHA1
```

Comment:
Sorry what I mean was, do we need this configuration in the application.yml that is bundled with the app at build time? Surely this is runtime configuration only and as such should be provided by the environment (via the syndesis-rest configmap in this case)? I don't like having stuff like encryption keys as defaults as it increases the chance of a publicly readable encryption key accidentally being used in a real deployment, e.g. if the configmap doesn't have the right configuration.